recipes/lammps/gh200/environments.yaml

```diff
@@ -0,0 +1,22 @@
+lammps:
+  compiler:
+  - toolchain: gcc
+    spec: gcc@12.3
+  mpi:
+    spec: cray-mpich@8.1.28
+    gpu: cuda
+  unify: true
+  specs:
+  - cuda@12
+  - kokkos@develop +wrapper
+  - lammps@develop
```

Comment:
If `develop` is really needed for both, can you pin to a specific commit instead with `@git.<COMMIT_HASH>`?

Reply:
i've removed develop I don't think it's needed. I had it at the time to quickly check the latest release performance on daint vs santis.